src/main/java/Duke.java

```diff
@@ -14,36 +16,66 @@ public static void main(String[] args) {
         System.out.println(greeting);
 
         while(true){
-            String input = sc.nextLine();
+            String userInput = sc.nextLine();
             System.out.println("----------------------------------------------");
-            if(input.equals("bye")){
+            if(userInput.equals("bye")){
                 System.out.println(bye);
                 break;
             }
-            else if(input.equals("list")){
+            else if(userInput.equals("list")){//show list
                 System.out.println("Here are the tasks in your list:");
```

Comment:
Perhaps you could wrap your list function in a method? Additionally, I think the comment can be indented relative to its position in the code (I noticed this in other parts of the code) :)